spring-kafka/src/test/java/org/springframework/kafka/support/DefaultKafkaHeaderMapperTests.java

```diff
@@ -48,6 +48,7 @@
  * @since 1.3
  *
  */
+@SuppressWarnings("deprecation")
```

Comment:
What deprecated API do we use in this test class?
Can we get rid of it?

Reply:
Oops - originally I was going to deprecate the raw strings properties, but they are also used by the SimpleKafkaHeaderMapper.